content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -287,6 +287,8 @@ From a shell on your instance, in the directory where you placed the `replicated
 1. The system will now perform a set of pre-flight checks on the instance and
    configuration thus far and indicate any failures. You can either fix the issues
    and re-run the checks, or ignore the warnings and proceed. If the system is running behind a proxy and is unable to connect to `releases.hashicorp.com:443`, it is likely safe to proceed; this check does not currently use the proxy. For any other issues, if you proceed despite the warnings, you are assuming the support responsibility.
+1. Set an encryption password used to encrypt the sensitive information at rest. The default value here is autogenerated but we suggest you enter your own password here.
```

Comment:
"The default value is auto-generated, but we strongly suggest you create your own password."